app/controllers/admin/custom_contents_controller.rb

```diff
@@ -8,8 +9,22 @@ def update
       params.require(:settings).map do |param|
         data = param.to_unsafe_h
         CustomSetting[data['var']] = data['value']
+        setting = CustomSetting.find_by(var: data['var']) || CustomSetting.new(var: data['var'])
```

Comment:
__[reek]__
* [DuplicateMethodCall] calls data['var'] 3 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)
* [FeatureEnvy] refers to data more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)

<!-- Comment by [SideCI](https://sideci.com) -->